Frontend/ui-customer/src/Components/QuoteListModel/View.purs

```diff
@@ -215,8 +216,90 @@ findingRidesView state push =
       , height $ V 300
       , width $ V 300
       ]
+  , addTipView state push
   ]
 
+addTipView :: forall w . QuoteListModelState -> (Action  -> Effect Unit) -> PrestoDOM (Effect Unit) w
```

Comment:
can you check if we can use the same popup modal we use in homescreen for tip view here as well to reduce code and actions redundancy

Reply:
No both are different we can not use same popup modal .